Source/ACE.DatLoader/Entity/AnimationHook.cs

```diff
@@ -0,0 +1,101 @@
+﻿using ACE.DatLoader.Entity.AnimationHooks;
+using ACE.Entity.Enum;
+
+namespace ACE.DatLoader.Entity
+{
+    public class AnimationHook
+    {
+        public AnimationHookType HookType { get; set; }
+        public uint Direction { get; set; }
+        private IHook _hook = null;
+        public IHook Hook { get { return _hook; } set { _hook = value; } }

```

Comment:
this should probably be "private set".  The reason being that this class should be the only place that hook value is ever _set_, and everybody else strictly reads it.